docs/release_notes/0.56.0.md

```diff
@@ -0,0 +1,13 @@
+# Release 0.56.0
+
+## Features
+- Configure parallel upgrades with UpdateConfig and MaxUnavailable (or MaxUnavailablePercentage) (#3753)
+- Bootstrap update nodegroup command (#3828)
+- Update the maxUnavailable number of nodes during parallel upgrades (#3855)
+- `Update` command can update multiple nodegroups (#3914)
+
+## Improvements
+
+- Add default tags to OIDC provider resources (#3885)
+- Do not set cgroupdriver to systemd in legacy bootstrap path (#3908)
```

Comment:
```suggestion
- Do not set cgroupdriver to systemd in legacy bootstrap path (#3908)
- Avoid cn-north-1d (#3920)
```

not merged yet but needs to go out in next rc

Reply:
ah sorry, I made a commit before seeing that you'd added this suggestion!